src/client/index.html

```diff
@@ -9,6 +9,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <!-- inject:css -->
   <!-- endinject -->
+  <script src="//cdnjs.cloudflare.com/ajax/libs/snap.svg/0.4.1/snap.svg-min.js"></script>
```

Comment:
This is typically a no-no, but since Snap.svg is going away anyways, it's not that big of a deal.